src/commands/games/sus.ts

```diff
@@ -0,0 +1,30 @@
+import { CommandInt } from "../interfaces/CommandInt";
+
+export const sus: CommandInt = {
+  prefix: "sus",
+  description: "Returns a color is sus",
+  parameters: "*none*",
+  command: async (message) => {
+    const array = [
+      "red",
+      "blue",
+      "green",
+      "pink",
+      "orange",
+      "yellow",
+      "black",
+      "white",
+      "purple",
+      "brown",
+      "cyan",
+      "lime",
+    ];
+    let i = Math.floor(Math.random() * array.length);
+    const color = array[i];
+    const embed = new MessageEmbed();
+    embed.setTitle(`Emergency Meeting!`);
+    embed.setDescription(`${color} is sus`);
+
+    message.channel.send(embed);
+  },
+};
```

Comment:
```suggestion
import CommandInt from "@Interfaces/CommandInt";
import { MessageEmbed } from "discord.js";

const sus: CommandInt = {
  name: "sus",
  description: "Returns a color is sus",
  run: async (message) => {
    const array = [
      "red",
      "blue",
      "green",
      "pink",
      "orange",
      "yellow",
      "black",
      "white",
      "purple",
      "brown",
      "cyan",
      "lime",
    ];
    let i = Math.floor(Math.random() * array.length);
    const color = array[i];
    const embed = new MessageEmbed();
    embed.setTitle(`Emergency Meeting!`);
    embed.setDescription(`${color} is sus`);

    message.channel.send(embed);
  },
};

export default sus;
```
Because you were working off the pre-refactor code, the CommandInt was outdated. Here's the fixed version.